tests/integration/test_sqs.py

```diff
@@ -2017,16 +2017,16 @@ def receive_message(attribute_names, message_attribute_names=None):
             )
 
         response = receive_message(["All"])
-        assert snapshot.match("all_attributes", response)
+        snapshot.match("all_attributes", response)
 
         response = receive_message(["All"], ["All"])
-        assert snapshot.match("all_system_and_message_attributes", response)
+        snapshot.match("all_system_and_message_attributes", response)
```

Comment:
at least that was my feedback, since the `assert` gives a clear indiciation in the test where an assertion is made. debatable though :shrug:

Reply:
The issue (imo) is that this is actually a bit misleading since the assert is actually quite far away from the match call.
@steffyP  and I also discussed if it therefore might make sense to rename `.match` to something else since the actual comparison & assertion is completely decoupled from the `.match` calls.